bin/release.sh

```diff
@@ -26,6 +26,20 @@ cd /app/bugsnag-js
 npm ci
 npm run bootstrap -- --ci
 
+# increment package version numbers
+if [ ! RETRY_PUBLISH ]; then
```

Comment:
[Spellcheck](https://www.shellcheck.net) doesn't like this:

> ^-- [SC2078](https://www.shellcheck.net/wiki/SC2078) (error): This expression is constant. Did you forget a $ somewhere?

```suggestion
if [ -z "$RETRY_PUBLISH" ]; then
```